lib/config-all.json

```diff
@@ -25,6 +25,7 @@
   "feature-detects": [
     "a/download",
     "ambientlight",
+    "mediastream",
```

Comment:
This list is alphabetically sorted, same comments as in the list below. This case it is after `"mediaquery/pointermq",`